packages/gatsby-theme-api-docs/src/components/type/type.js

```diff
@@ -87,6 +84,7 @@ const ApiType = (props) => {
 
 ApiType.propTypes = {
   apiKey: PropTypes.string.isRequired,
+  apiTypes: PropTypes.object.isRequired,
```

Comment:
Isn't this an array?

```suggestion
  apiTypes: PropTypes.array.isRequired,
```

Nit: even better is to specify the shape of the array

Reply:
thanks! Fixed that wrong definition. 

specifying the shape of the array would get pretty verbose, so I avoided that.  It#s the result of this query: https://github.com/commercetools/commercetools-docs-kit/blob/main/packages/gatsby-theme-api-docs/src/hooks/use-api-types.js#L11